src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/containers/EntityFluidContainerProvider.java

```diff
@@ -0,0 +1,15 @@
+package io.github.fablabsmc.fablabs.api.fluidvolume.v1.containers;
+
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api.FluidContainer;
+
+import net.minecraft.util.math.Direction;
+
+/**
+ * an entity that carries fluids.
+ */
+public interface EntityFluidContainerProvider {
+	/**
+	 * @return the fluid container for this entity
+	 */
+	FluidContainer getContainer(Direction face);
```

Comment:
what's the face of an entity?

Reply:
The direction the entity fluid container is being accessed from?

Reply:
what i5 said, will make sure to include that